downstream/modules/devtools/proc-devtools-install-container.adoc

```diff
@@ -2,14 +2,13 @@
 
 = Installing {ToolsName} on a container inside {VSCode}
 
-
 The Dev Containers {VSCode} extension requires a config file to store settings for your dev containers. 
```

Comment:
Suggestion: "Dev Containers {VSCode}" sounds like a UI element that could be wrapped in bold.